src/memote/support/consistency.py

```diff
@@ -219,23 +217,28 @@ def find_inconsistent_min_stoichiometry(model, atol=1e-13):
     metabolites = sorted(internal_mets, key=get_id)
     stoich, met_index, rxn_index = con_helpers.stoichiometry_matrix(
         metabolites, reactions)
-    left_ns = con_helpers.nullspace(stoich.T)
-    # deal with numerical instabilities
-    left_ns[np.abs(left_ns) < atol] = 0.0
-    LOGGER.info("nullspace has dimension %d", left_ns.shape[1])
+    left_ns = con_helpers.nullspace(stoich.T, atol)
+    if len(left_ns.shape) > 1:
```

Comment:
We decided that if the nullspace is empty, we can return early with a list of every unconserved metabolite in a singleton set.